awx/conf/settings.py

```diff
@@ -136,6 +138,14 @@ def filter_sensitive(registry, key, value):
     return value
 
 
+# settings.__getattr__ is called *constantly*, and the LOG_AGGREGATOR_ ones are
+# so ubiquitous when external logging is enabled that they should kept in memory
+# with a short TTL to avoid even having to contact memcached
+# the primary use case for this optimization is the callback receiver
+# when external logging is enabled
+LOGGING_SETTINGS_CACHE = cachetools.TTLCache(maxsize=50, ttl=1)
```

Comment:
@AlanCoding I decided to change this back to 1 second, because I'm still seeing drastic speed improvements with 1 second, and it's likely to much less noticeable for users.

Reply:
well, no need to document _that_